Assets/Scripts/Manager/StorySceneManager.cs

```diff
@@ -0,0 +1,577 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+using System.Collections.Generic;
+
+public class StorySceneManager : MonoBehaviour {
```

Comment:
I'm gonna let this slide now cause we're on a time crunch, but next time design wise I would recommend splittling this to different files. 
Each part of the scene can be it's own class that way it's easier to understand the flow of things. You can even create an abstract class for variables that are common among each scene.